runtime/doc/lsp.txt

```diff
@@ -377,6 +377,11 @@ LSP HIGHLIGHT                                                    *lsp-highlight*
 
 Reference Highlights:
 
+Highlight groups that are meant to be used by |vim.lsp.buf.document_highlight()|.
+
+You can see more about the differences in types here:
```

Comment:
What do you mean by `types`?

Reply:
Well for document highlights there are three different types: 
- Text
- Read
- Write

They are explained in the given link.

Reply:
maybe `document highlight types` for clarity, but I leave it to you if that is overly verbose

Reply:
eh, I think it's sort of implied since that's what the section is talking about.